system-contracts/contracts/precompiles/EcAdd.yul

```diff
@@ -22,8 +22,8 @@ object "EcAdd" {
             ///       needed to execute one instance of the corresponding precompile in the circuit.
             function ECADD_GAS_COST() -> ret {
                 // Current geometry: cycles_per_ecadd_circuit: 812,
-                // so 80'000 / 812 == 98
-                ret := 98
+                // so 80'000 / 752 == 107
```

Comment:
please update the comment above as well

Reply:
what does 80k mean in this context? Total gas per circuit and then we divide by the number of operations of this type we can fit in a circuit?

Reply:
yes